erpnext/accounts/report/general_ledger/general_ledger.py

```diff
@@ -128,6 +128,10 @@ def get_gl_entries(filters):
 		filters['company_fb'] = frappe.db.get_value("Company",
 			filters.get("company"), 'default_finance_book')
 
+	dim_fields = ""
```

Comment:
```suggestion
	dimension_fields = "cost_center"
```

Reply:
Renamed the variable, but I don't see how it can be initialized to "cost_center". The existing code handles cost_center separately and quite specially. I think it would be great if ERPNext was streamlined to have cost center just precisely be an accounting dimension that is present by default, but I feel the reality is that the code is far from that now and I don't see how this small PR can make any meaningful step toward true integration of cost center as an accounting dimension.